src/components/connect/Company.vue

```diff
@@ -38,21 +38,6 @@ export default {
 
 <style lang="scss" scoped>
 @import "~assets/styles/vars";
-// #container {
```

Comment:
Does it make sense to squash 7da97c5 and 82c4679 (this one)? The prior commit comments out these CSS blocks. But they end up being deleted only a commit later here.